relay-server/src/services/project.rs

```diff
@@ -691,8 +694,17 @@ impl Project {
         outcome_aggregator: Addr<TrackOutcome>,
         envelope_processor: Addr<EnvelopeProcessor>,
         buckets: Vec<Bucket>,
+        project_cache: Addr<ProjectCache>,
     ) {
-        if self.metrics_allowed() {
+        if self.config.relay_mode() == RelayMode::Proxy {
+            if let Some(scoping) = self.partial_scoping() {
+                let mut bucket_map = HashMap::new();
+                bucket_map.insert(scoping.project_key, buckets);
+                project_cache.send(FlushBuckets {
```

Comment:
skipped the aggregator for simplicity because it checks for scoping from projectstate and also features in `rate_limit_and_merge_buckets`. 
 
i wasnt sure if we should also skip `handle_flush_buckets`? it checks if it's ratelimited, do we care about ratelimiting in proxy mode?